docs/best_practice/crate_ami.txt

```diff
@@ -0,0 +1,114 @@
+.. highlight:: yaml
+.. _crate_ami:
+
+================================
+Launch a custom Crate AMI on AWS
+================================
+
+Crate provides custom AMIs (see `AMI`_) that can be launched from every
+EC2 user by using the AWS Management Console or `AWS CLI`_. The naming
+convention is as follows:
+
+  ``Crate-AWS-AMI-<VERSION>-<REVISION>``
```

Comment:
maybe add: ... where VERSION is the Crate version in format `x.x.x` and revision is the build revision.